GiEnJul/ClientApp/src/components/InputFields/SelectForm.tsx

```diff
@@ -45,37 +77,22 @@ const SelectForm: FC<ISelect> = ({
       disabled={disabled}
       fullWidth={fullWidth}
       margin={margin}
+      focused={!disabled && isMob}
     >
-      <InputLabel htmlFor={id}>{label}</InputLabel>
+      <InputLabel variant={variant} htmlFor={id}>
+        {label}
+      </InputLabel>
       <Select
         native={isMob}
         inputProps={{
           name: name,
           id: id,
         }}
         label={label}
+        value={value}
         {...rest}
       >
-        {options &&
-          options.map((o) =>
-            isMob ? (
-              <option
-                key={`n_${name}_${o.text}`}
-                value={o.value}
-                className="text-capitalize"
-              >
-                {o.text}
-              </option>
-            ) : (
-              <MenuItem
-                key={`${name}_${o.text}`}
-                value={o.value}
-                className="text-capitalize"
```

Comment:
Small thing but the "text-capitalize" makes the first letter in each word to upper case. Would suggest to remove line 65 and73.
![image](https://user-images.githubusercontent.com/37541100/127163972-f47783c0-f53e-408f-94a3-70ccfbb3d15a.png)